.github/workflows/ci-nix.yml

```
@@ -0,0 +1,20 @@
+name: Nix
+on:
+  push:
+    branches:
+      - master
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2.3.4
+      - uses: cachix/install-nix-action@v12
+        with:
+          install_url: https://github.com/numtide/nix-flakes-installer/releases/download/nix-2.4pre20210429_d15a196/install
+      - name: Nix Config
+        run: |
+          mkdir -p ~/.config/nix/
+          echo '
+            experimental-features = nix-command flakes' > ~/.config/nix/nix.conf
```

```suggestion
      - uses: cachix/install-nix-action@v17
```

Will install the latest Nix, flakes-enabled.

---

I put the newer version.